### ISSUE TITLE
Add an "Edit this page" link to each page

### ISSUE THREAD

**Issue by LenzGr** [westerndigitalcorporation/zonedstorage.io#49]
Having a quick and direct way to make small changes to a web page's content further lowers the barrier for getting improvement suggestions to the documentation and web site. Have you considered adding a feature that provide an "Edit" option that would open the Markdown file on github in Edit mode?

**Comment by zdover23** [comment]
Lenz,

I think this is a fine suggestion, but I also think that there should be a English-grammar-savvy gatekeeper who has veto power over any suggestions made through such a channel as the one you propose. I realize in typing this out just how self-serving this suggestion might seem.

**Comment by damien-lemoal** [comment]
> Having a quick and direct way to make small changes to a web page's content further lowers the barrier for getting improvement suggestions to the documentation and web site. Have you considered adding a feature that provide an "Edit" option that would open the Markdown file on github in Edit mode?

Well, "easy" is all relative... Personally, I find working with git cli + vim edit files far easier than any of the obscure GitHub web interface :)

Docusaurus has a "edit page" button but I disabled it for now. I need to have a look at how that is working and what happens in github when it is being used. Based on that, we can probably enable it, as long as changes come in the form of PRs that can be reviewed.

Will try to have a look later today or tomorrow.

**Comment by LenzGr** [comment]
> I think this is a fine suggestion, but I also think that there should be a English-grammar-savvy gatekeeper who has veto power over any suggestions made through such a channel as the one you propose.

Fair point, but reviewing and approving a pull request prior to merging it should be best practice anyway, no?

**Comment by damien-lemoal** [comment]
> > Having a quick and direct way to make small changes to a web page's content further lowers the barrier for getting improvement suggestions to the documentation and web site. Have you considered adding a feature that provide an "Edit" option that would open the Markdown file on github in Edit mode?
> 
> Well, "easy" is all relative... Personally, I find working with git cli + vim edit files far easier than any of the obscure GitHub web interface :)
> 
> Docusaurus has a "edit page" button but I disabled it for now. I need to have a look at how that is working and what happens in github when it is being used. Based on that, we can probably enable it, as long as changes come in the form of PRs that can be reviewed.
> 
> Will try to have a look later today or tomorrow.

Late with this. Will try harder to have a look :)
